.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Bug Report
+about: Create a detailed report about a deficiency in the BitShares Core implementation.
+
+---
+
+**Instructions**
+Pleaae include a detailed Title above. Next, please complete the following sections below:
+* Bug Description
+* Steps To Reproduce
+* Expected Behavior
+* Screenshots (optional)
+* witness_node --version (optional)
+* Aditional Context (optional)
+
+Finally, press the 'Submit new issue' button. The Core Team will evaluate and prioritize your Bug Report for future development. 
+
+**Bug Description**
+A clear and concise description of what the bug is.
+
+**Steps To Reproduce**
+Steps to reproduce the behavior:
+1. Execute API call '...'
+2. Using JSON payload '...'
+3. Received response '...'
+4. See error
+
+**Expected Behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain process flow and behavior.
+
+**witness_node --version (please complete the following information):**
+ - Version: [e.g. 2.0.180425]
+ - SHA: [e.g. bf263b1db47a91ea9748118df1abb5a22cafc40b]
+ - SSL: [e.g. OpenSSL 1.0.2g  1 Mar 2016]
+ - Boost: [e.g. 1.62]
+ - Websocket++: [e.g. 0.7.0]
```

Comment:
Websocketpp is a submodule in our sources, system library version doesn't matter.